test/unit/repo/MockitoPassByNameHelper.scala

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
This tech debt ticket introduces more tech debt - my view is better to fix that more tech debt in this story